allan_variance.py

```diff
@@ -120,3 +129,23 @@ def params_from_avar(tau, avar):
     prediction = A.dot(x) * avar
 
     return np.sqrt(x), prediction
+
+
+def allan_deviation(avar):
```

Comment:
I don't know, I'm not a fan. If you want a proper function, you have to duplicate the whole docstring of the `allan_variance` adding small changes. Having such duplicate function is annoying for maintenance --- you need to keep their docstrings in sync, so it sort of violates the DRY principle.

As for the usage:``adev = allan_variance(...) ** 0.5`` looks perfectly logical and easy to achieve. I think `allan_deviation` must do a little bit more (more than 1 line) to be worth including.